docker/importer/volume/fhirConfig.js

```diff
@@ -0,0 +1,101 @@
+'use strict'
+
+const fs = require('fs')
+const https = require('https')
+const http = require('http')
+const path = require('path')
+const { spawn } = require('child_process')
+
+const resourcesPath = path.resolve(__dirname, 'fhirResources')
+
+const downloadResources = (cbk) => {
+  const COVID19_IG_URL = process.env.COVID19_IG_URL || 'https://openhie.github.io/covid-ig'
+  
+  if (fs.existsSync())
+    fs.rmdirSync(resourcesPath)
+  
+  fs.mkdirSync(resourcesPath)
+  
+  const resources = fs.createWriteStream(path.resolve(resourcesPath, 'fhir-resources.zip'))
```

Comment:
Why do we need the data written into files? You just read the files in the next step? Why not just hold them in memory.